src/test/java/tr/com/srdc/cda2fhir/IntegrationTest.java

```diff
@@ -0,0 +1,167 @@
+package tr.com.srdc.cda2fhir;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Bundle.BundleEntryComponent;
+import org.hl7.fhir.dstu3.model.Bundle.BundleEntryResponseComponent;
+import org.hl7.fhir.dstu3.model.Bundle.BundleType;
+import org.hl7.fhir.dstu3.model.Bundle.BundleTypeEnumFactory;
+import org.hl7.fhir.dstu3.model.Patient;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import org.openhealthtools.mdht.uml.cda.ClinicalDocument;
+import org.openhealthtools.mdht.uml.cda.util.CDAUtil;
+
+import com.palantir.docker.compose.DockerComposeRule;
+import com.palantir.docker.compose.connection.waiting.HealthChecks;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.api.MethodOutcome;
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+import ca.uhn.fhir.validation.FhirValidator;
+import ca.uhn.fhir.validation.ValidationResult;
+import tr.com.srdc.cda2fhir.transform.CCDTransformerImpl;
+import tr.com.srdc.cda2fhir.util.FHIRUtil;
+import tr.com.srdc.cda2fhir.util.IdGeneratorEnum;
+
+public class IntegrationTest{
+	static String hapiURL = "http://localhost:8080";
+	static String serverBase = hapiURL + "/baseDstu3";
+	static BundleTypeEnumFactory bundleTypeEnumFactory;
+	static FhirContext ctx;
+	static IGenericClient client;
+	static CCDTransformerImpl ccdTransformer;
+	
+	@BeforeClass
+    public static void init() throws IOException {
+		
+        // Load MDHT CDA packages. Otherwise ContinuityOfCareDocument and similar documents will not be recognised.
+        // This has to be called before loading the document; otherwise will have no effect.
+        CDAUtil.loadPackages();
+        ctx = FhirContext.forDstu3();
+        bundleTypeEnumFactory = new BundleTypeEnumFactory();
```

Comment:
Not sure why you need this.  Can't you just use BundleType.TRANSACTION or other?

Reply:
Good Point.